Formula/rocksdb.rb

```diff
@@ -22,15 +22,21 @@ class Rocksdb < Formula
   def install
     ENV.cxx11
     ENV["PORTABLE"] = "1" if build.bottle?
-    if build.with? "lite"
-      ENV.append_to_cflags "-DROCKSDB_LITE=1"
-      ENV["LIBNAME"] = "librocksdb_lite"
-    end
-    system "make", "clean"
```

Comment:
Are the build directories ever re-used? On a fresh build `make clean` shouldn't be necessary.